crates/symbolicator/src/types/mod.rs

```diff
@@ -236,48 +235,38 @@ pub struct RawFrame {
     pub post_context: Vec<String>,
 
     /// Information about how the raw frame was created.
-    #[serde(
-        default,
-        with = "frame_trust",
-        skip_serializing_if = "is_default_value"
-    )]
+    #[serde(default, skip_serializing_if = "is_default_value")]
     pub trust: FrameTrust,
 }
 
-mod frame_trust {
-    use super::*;
-    use serde::{Deserializer, Serializer};
+#[derive(Debug, Copy, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum FrameTrust {
+    None,
+    Scan,
+    CfiScan,
+    Fp,
+    Cfi,
```

Comment:
Nitpicky, but I think the original names (`FramePointer`, `CallFrameInfo`) are better than their new ones here, despite their verbosity. 

Figuring out what these expand to on your own via the codebase is already difficult enough: there're only two places that mention what CFI expands out to, and one place that does the same for FP. The `From` implementation below serves as one of those spots because `rust-minidump` uses the full name. Looking these up via a search engine does not provide useful results, even if you add "frametrust" to the search query. This additional lookup to see what the acronym expands out to can be easily resolved by just using the original values. 

Using the acronyms in these variants makes it less unwelcoming for newcomers, and creates another place in the code where the easiest way to find an answer to one's question is to speak to a member of the project which we should really cut down on.